osgp/protocol-adapter-oslp/web-device-simulator/src/main/java/org/opensmartgridplatform/webdevicesimulator/service/OslpChannelHandler.java

```diff
@@ -1122,6 +1039,25 @@ private void handleSetTransitionRequest(final Device device, final SetTransition
         }
     }
 
+    private void sendEvent(final Device device, final Oslp.Event event, final String description) {
+        this.sendEventWithCustomDelay(device, event, description, 3000);
+    }
+
+    private void sendEventWithCustomDelay(final Device device, final Oslp.Event event, final String description,
+            final int delay) {
+        // Send an event.
```

Comment:
Opinions may differ, so I won't go on about this if it is left in, but I am no fan of comments that don't add anything that isn't already obvious from method names or other things in the code.
For me this looks like the category of comments to be avoided like: `x = x + 1; // increment x by 1`.

Reply:
I removed the useless comment.